ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -198,6 +198,9 @@ message PartyDetails {
   ObjectMeta local_metadata = 4;
 
   // The id of the ``Identity Provider``
-  // Optional, if not set, assume the party is managed by the default identity provider or party is not hosted by the participant.
+  // Optional, if not set, there could be 3 options:
+  // 1) the party is managed by the default identity provider.
+  // 2) party is not hosted by the participant.
+  // 3) party is hosted by the participant, but is outside of the user's identity provider.
   string identity_provider_id = 5;
```

Comment:
> This might be a radical change at this point but if we were to represent the default idp as a non-empty string (e.g. "default") then we could simplify this to say 2) and 3) only (i.e. we wouldn't have to mention 1) separately)

Agreed offline we wont be doing it.

> Also, if you're the an admin in the default idp you won't be able to tell whether a party is yours or not because in all cases this field is set to the empty string, correct?

Correct, but not sure if this is what we want. `participant_admin` may lookup all known IDPs to this participant and get parties by which IDP then. This way it would be possible for him to learn all "participant-local" IDPs.

> Does bool is_local = 3;'s description needs an update to to say what happens if the party is hosted by in a different idp?

Will add a comment there too